<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>path_viewer</class>
 <widget class="QWidget" name="path_viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>251</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="vertical_layout">
   <item>
    <widget class="QSplitter" name="main_splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="ues::gui::path_view" name="viewer_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QWidget" name="control_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="control_vertical_layout">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QFormLayout" name="control_form_layout">
          <item row="2" column="1">
           <widget class="QComboBox" name="scenario_combo_box">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>&lt;No available scenarios&gt;</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="scenario_label">
            <property name="text">
             <string>Scenario:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="paths_widget" native="true">
         <layout class="QVBoxLayout" name="paths_vertical_layout">
          <item>
           <widget class="QCheckBox" name="reference_checkbox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Show edge sampling path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="approximate_checkbox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Show approximate path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="approximate_filtering_checkbox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Show approximate path with filtering</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="plane_cut_checkbox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Show plane cut path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="rrt_checkbox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Show RRT path</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="prm_checkbox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Show PRM path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="bitstar_checkbox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Show BIT* path</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="vertical_spacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="save_image_button">
         <property name="text">
          <string>Save to image</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ues::gui::path_view</class>
   <extends>QWidget</extends>
   <header>graphics/path_view.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
